[ALL] Feat: main 예약 상태 확인 기능 구현

yongsuson opened the pull request:
[ALL] Feat: main 예약 상태 확인 기능 구현

yongsuson committed:
commit

commit

yongsuson committed:
[ALL] Feat: main 예약 상태 확인 기능 구현

[ALL] Feat: main 예약 상태 확인 기능 구현

umyonghyun commented on src/main/java/com/eungaehospital/appointment/controller/AppointmentViewController.java:
```diff
@@ -0,0 +1,10 @@
+
+@Controller
+@RequiredArgsConstructor
+public class AppointmentViewController {
```
삭제 필요

umyonghyun commented on src/main/java/com/eungaehospital/appointment/dto/AppointmentResponseDto.java:
```diff
@@ -0,0 +1,31 @@
+import lombok.Data;
+
+@Builder
+@Data
```
Getter, Setter, RequiredArgsConstructor, ToString, EqualsAndHashCode,
EqualsAndHashCode ->어느곳에서 사용되는 건지 찾아보기 @yongsuson , @seanlee0923 , @jihye525 , @ghktndyd , @ooojin000

umyonghyun replied:
@jun9898

umyonghyun commented on src/main/java/com/eungaehospital/appointment/service/AppointmentService.java:
```diff
@@ -0,0 +1,43 @@
+		List<Appointment> appointment = appointmentRepository
+			.getAppointmentByAppointmentDateAndHospitalId(
+				hospitalSeq,
+				convertStringToLocalDate(selectDate));
```
Spring converter 사용 formatter

yongsuson committed:
[FE] Feat: 예약 없을 시 예약 없음 기능 추가

[FE] Feat: 예약 없을 시 예약 없음 기능 추가, 나이 --> 생년월일로 표시 변경, AppointmentViewController 삭제
